api/prometheus/v1/api.go

```diff
@@ -56,12 +56,12 @@ type HealthStatus string
 const (
 	// Possible values for ErrorType.
 	ErrBadData     ErrorType = "bad_data"
-	ErrTimeout               = "timeout"
-	ErrCanceled              = "canceled"
-	ErrExec                  = "execution"
-	ErrBadResponse           = "bad_response"
-	ErrServer                = "server_error"
-	ErrClient                = "client_error"
+	ErrTimeout     ErrorType = "timeout"
+	ErrCanceled    ErrorType = "canceled"
+	ErrExec        ErrorType = "execution"
+	ErrBadResponse ErrorType = "bad_response"
+	ErrServer      ErrorType = "server_error"
+	ErrClient      ErrorType = "client_error"
```

Comment:
Where is this change coming from? Is it needed/intended?

Reply:
It was spotted by the newer version of staticcheck that is compatible with Go modules:

```
api/prometheus/v1/api.go:58:2: only the first constant in this group has an explicit type (SA9004)
```

https://github.com/dominikh/go-tools/blob/master/cmd/staticcheck/docs/checks/SA9004

Reply:
Ah cool, thanks.